build/contracts/ConvertLib.json

```diff
@@ -1,5 +1,5 @@
 {
-  "contract_name": "ConvertLib",
+  "contractName": "ConvertLib",
```

Comment:
What's the reason for the name change?

Reply:
"truffle compile" command make these changes